src/datatypes/uncertainty.coffee

```diff
@@ -11,6 +11,10 @@ module.exports.Uncertainty = class Uncertainty
         # Uncertainties with different types probably shouldn't be supported.
         return false
       if typeof a.after is 'function' then a.after b else a > b
+    isCode = (val) ->
```

Comment:
I think we should follow the recent pattern in `isQuantity` or `isDatetime` and add this to `src/datatypes/clinical.coffee`:

```coffeescript
  # Define a simple getter to allow type-checking of this class without instanceof
  # and in a way that survives minification (as opposed to checking constructor.name)
  Object.defineProperties @prototype,
    isCode:
      get: -> true
```

Then this logic would change to:
```coffeescript
if @low.isCode then @low = null
if @high.isCode then @high = null
```

Reply:
@losborne makes a great point -- and I don't think it should be hard to implement (given he more or less included the implementation in his comment).  So... I'd suggest we follow his advice.

Reply:
@mayerm94 mentioned in a private thread that we might also need to do the same for `ValueSet` and `Concept`.  I'm good with that as well!